dask_expr/reductions.py

```diff
@@ -408,7 +368,49 @@ def aggregate_kwargs(self):
         return {"dropna": self.dropna}
 
 
-class ValueCounts(Reduction):
+class ReductionConstantDim(Reduction):
+    @classmethod
+    def chunk(cls, df, **kwargs):
+        return cls.reduction_chunk(df, **kwargs)
+
+    @classmethod
+    def combine(cls, inputs: list, **kwargs):
+        func = cls.reduction_combine or cls.reduction_aggregate or cls.reduction_chunk
+        df = _concat(inputs)
+        return func(df, **kwargs)
```

Comment:
If I understand correctly, the only difference is that we don't want to convert a series back to dataframe.
Can we just add an `enforce_dataframe = True` class property to `Reduction`, and just set it to `False` for `ValueCounts`, etc?

Reply:
Personally, I find this more readable and clear than a flag that's buried in the functions, but no strong opinion. Happy to change if you prefer

A flag should do the trick, yes

Reply:
Ah, that's fair. In hindsight, I think I just find the `ReductionConstantDim` name a bit confusing. Could this be `SeriesReduction`? Or, am I misunderstanding the meaning of `ReductionConstantDim`?

Reply:
Maybe the name is not ideal, this means that you reduce the number of rows but the dimension stays the same. Nlargest for example reduces the number of rows in your DataFrame but the result will still be a DataFrame, while sum reduces to a Series

Reply:
Okay, I see. Maybe just add a description in the doc-string and I'll be fine with the name then.

Reply:
Added a docstring